app/src/main/java/com/example/android/sunshine/app/ForecastAdapter.java

```diff
@@ -14,63 +13,26 @@
 import com.bignerdranch.android.multiselector.SingleSelector;
 import com.bignerdranch.android.multiselector.SwappingHolder;
 import com.bumptech.glide.Glide;
-import com.example.android.sunshine.app.data.WeatherContract;
+import com.example.android.sunshine.app.data.WeatherConditions;
+import com.example.android.sunshine.app.utils.DateUtility;
+import com.example.android.sunshine.app.utils.PrefUtility;
+import com.example.android.sunshine.app.utils.WeatherUtility;
 
+import java.util.ArrayList;
 import java.util.List;
 
-/**
- * {@link ForecastAdapter} exposes a list of weather forecasts
- * from a {@link Cursor} to a {@link RecyclerView}.
- */
 public class ForecastAdapter extends RecyclerView.Adapter<ForecastAdapter.ViewHolder> {
 
-    private static final int VIEW_TYPE_TODAY = 0;
-    private static final int VIEW_TYPE_FUTURE_DAY = 1;
+    private static final int VIEW_TYPE_TODAY = R.layout.list_item_forecast_today;
+    private static final int VIEW_TYPE_FUTURE_DAY = R.layout.list_item_forecast;
```

Comment:
This is a neat trick that I learnt from Yigit Boyar. So basically, I use the layout id reference as unique ints, but at the same time I get for free the layout id that I will have to inflate in `onCreateViewHolder()` for the respective view type.